ipset_linux.go

```diff
@@ -401,9 +414,11 @@ func (result *IPSetResult) parseAttrData(data []byte) {
 		case nl.IPSET_ATTR_IP | nl.NLA_F_NESTED:
 			for nested := range nl.ParseAttributes(attr.Value) {
 				switch nested.Type {
-				case nl.IPSET_ATTR_IP | nl.NLA_F_NET_BYTEORDER:
+				case nl.IPSET_ATTR_IPV4 | nl.NLA_F_NET_BYTEORDER: // fallthrough
+				case nl.IPSET_ATTR_IPV6 | nl.NLA_F_NET_BYTEORDER:
 					result.Entries = append(result.Entries, IPSetEntry{IP: nested.Value})
-				case nl.IPSET_ATTR_IP:
+				case nl.IPSET_ATTR_IPV4: // fallthrough
```

Comment:
if the goal is for these cases to fallthrough, in go they need an explicit fallthrough statement: https://golang.org/ref/spec#Switch_statements

Reply:
Oh, I really dont remember what I was thinking here..

Should be fixed now.